### PR TITLE
title component v1

### PR DESCRIPTION
**Pull request by jordosgoite**
😝

### CODE REVIEW

File: components/title/title.jsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+import styles from './title.module.scss';
+
+/**
+ * @typedef {Object} TitleProps
+ * @property {string} text - the text that would be in the title
+ * @property {string} [emoji] - emoji to animate the title
+ */
+
+/**
+ *
```
Comment (diegonzs) [inline components/title/title.jsx]: Add a description for the component.

File: components/title/title.jsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+ */
+
+/**
+ * 
+ * @param {TitleProps} props
+ */
+
+export const Title = ({ text, emoji }) => {
+  return (
+    <h1 className={styles.title}>{text} {emoji}</h1>
```
Comment (diegonzs) [inline components/title/title.jsx]: Wrapped the emoji with a span an added a class to style it. in the xd it has a separation with the text of 24px.

File: components/title/title.jsx [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+Title.propTypes = {
+    /** the text that would be in the title */
+    text: PropTypes.string.isRequired,
+    /** Emoji to animate the title */
+    emoji: PropTypes.string,
+  }
+
+Title.defaultProps ={
+    text: "WHAT'S HAPPENING",
+  }
```
Comment (diegonzs) [inline components/title/title.jsx]: Removed default props because text is required 😅

File: components/title/title.module.scss [hunk 4]
```diff
@@ -0,0 +1,5 @@
+@import 'variables.scss';
+.title{
+  color: $color-text-95;
+  font-size: $font-size-huge;
+}
```
Comment (diegonzs) [inline components/title/title.module.scss:8]: Create the class emoji to add those 24px of margin left to separate the text and the emoji.

**Review by diegonzs — CHANGES_REQUESTED** [review]
Fixed comments I left in the code.

File: components/title/title.jsx (hunk 1 above)
Comment (jordosgoite) [reply components/title/title.jsx]: Done

File: components/title/title.jsx (hunk 2 above)
Comment (jordosgoite) [reply components/title/title.jsx]: Done!

File: components/title/title.jsx (hunk 3 above)
Comment (jordosgoite) [reply components/title/title.jsx]: Done!

File: components/title/title.module.scss (hunk 4 above)
Comment (jordosgoite) [reply components/title/title.module.scss:8]: Done!

**Comment by jordosgoite** [comment]
modified according to suggestions

**Review by diegonzs — APPROVED** [review]

**Pull request merged**